todo_app/data/Items.py

```diff
@@ -0,0 +1,5 @@
+class ToDo:
+  def __init__(self, id, status, title):
+    self.id = id
+    self.status = status
+	self.title = title
```

Comment:
The indenting for this line doesn't look quite right, and it looks like we never use this class - we should convert our trello json responses with the card details into ToDo objects, and use those in our templates